google/cloud/bigtable/backup.py

```diff
@@ -0,0 +1,392 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A user-friendly container for Cloud Bigtable Backup."""
```

Comment:
not sure container is the correct word here? wrapper?. Also `a Cloud Bigtable Backup` or `Cloud Bigtable Backups` perhaps?  

And it seems we mix `google cloud bigtable` and `cloud bigtable`. Maybe best to be consistent?

Reply:
Agreed about the `wrapper` vs `container` as well as `Google Cloud Bigtable`. Not sure about the article though, since it's a definition of notion here, rather than "a" (i.e. particular) Backup class. I'll put it in for now, just must say that it's debatable.